### PR TITLE
first impl with localstack (stack of AWS services running in Docker)

### PR DESCRIPTION
**Pull request by aesteve**
What does Travis has to say ?

### CODE REVIEW

**Comment by vietj** [comment]
it would be good to split concerns (like Junit5 and the test one) in separate PR :-)

**Comment by vietj** [comment]
Travis seems happy!

**Comment by aesteve** [comment]
Yeah, I should have merged the Junit5 stuff before, from another PR.

I needed it for multiple ` @ExtendWith` and not a simple `@RunWIth(...)`

File: src/test/java/io/vertx/ext/awssdk/AsyncHttpClientTest.java [hunk 1]
```diff
@@ -33,32 +40,32 @@
...
+      server.close(res -> {
+          assertTrue(res.succeeded());
+          ctx.completeNow();
+      });
   }
 
   @Test
-  public void testGet(TestContext ctx) {
-    Async async = ctx.async();
+  @Timeout(value = 5, timeUnit = TimeUnit.SECONDS)
```
Comment (vietj) [inline src/test/java/io/vertx/ext/awssdk/AsyncHttpClientTest.java]: such low timeout might be an issue on travis CI that runs slower than on your laptop

**Pull request merged**